static/docs/use-cases/using-shared-server-for-the-team.md

```diff
@@ -0,0 +1,94 @@
+# Using Shared Server For The Team
+
+The key principle that every data science team should aim for is to make
```

Comment:
Let's try to start with the description of the scenario first:

It's pretty common to see that teams prefer using one single shared machine to run their experiments ...

Reply:
Then, we need to mention briefly, why is is happening:

* Better resource utilization - I can utilize multiple GPUs, for example
* Probably, less expensive
* Data locality
* something I am missing?

Reply:
Ok